### PR TITLE
[SRVCOM-1699] Updates to 1.31 release notes

### PR DESCRIPTION
**Pull request by rh-max**
Version(s):
`serverless-docs-1.30`+

Issue:
https://issues.redhat.com/browse/SRVCOM-1699 , direct feedback from Product Owner

Link to docs preview:
fixme

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Comment by rh-max** [comment]
@maschmid Hi Marek, could you PTAL? Thanks.

File: modules/serverless-tech-preview-features.adoc [hunk 1]
```diff
@@ -15,15 +15,15 @@ The following table provides information about which {ServerlessProductName} fea
...
 |-
-|TP
+|GA
 
-|Golang functions
+|Using {SMProductShortName} to isolate network traffic with {ServerlessProductShortName}
 |-
 |-
-|TP
+|GA
```
Comment (maschmid) [inline modules/serverless-tech-preview-features.adoc:26]:
```suggestion
|TP
```

If the "|Using {SMProductShortName} to isolate network traffic with {ServerlessProductShortName}" refers to 
"{ServerlessProductName} multi-tenancy with {SMProductShortName} is now available as a Technology Preview (TP) feature.",  it should be marked as TP

**Comment by rh-max** [comment]
Thanks @maschmid !

**Comment by maschmid** [comment]
QE approved

**Comment by maschmid** [comment]
(assuming the {sno} expands properly, the preview is not available yet)

**Review by sheriff-rh — COMMENTED** [review]
No build preview, but this seems like a logical change, so LGTM. 

Please squash your commits @rh-max and let me know when you're ready to merge.

**Comment by sheriff-rh** [comment]
Build passed; merging.

**Pull request merged**